tests/UATests.php

```diff
@@ -37,7 +37,7 @@ public function testBots()
 
         foreach ($lines as $line) {
             $test = Crawler::isCrawler($line);
-            $this->assertEquals($test, true, $line);
+            $this->assertTrue($test);
```

Comment:
Can we keep the `$line` var please as the second parameter, so we know what `string` the test fails on...

```php
$this->assertTrue($test, $line);
```